projectional/src/main/java/jetbrains/jetpad/projectional/cell/BaseProjectionalSynchronizer.java

```diff
@@ -649,4 +724,28 @@ public int size() {
       return myRoleSynchronizer.getMappers().size();
     }
   }
+
+  private class ContentToItemMapper<ContentT> {
+    private Function<ContentT, SourceItemT> myFromContent;
+    private Function<SourceItemT, ContentT> myToContent;
```

Comment:
Why do you have a method for accessing myFromContent but not for myToContent? myToContent looks unused at first glance, you have to search for usages to see that it's actually used.